sql/handler.h

```diff
@@ -1424,6 +1424,21 @@ struct handlerton
    int  (*recover)(handlerton *hton, XID *xid_list, uint len);
    int  (*commit_by_xid)(handlerton *hton, XID *xid);
    int  (*rollback_by_xid)(handlerton *hton, XID *xid);
+   /*
+     recover_rollback_by_xid is optional. If set, it will be called instead
+     of rollback_by_xid when transactions should be rollback at server startup.
+
+     This function should just changed the transaction's state from prepared to
+     active, and rollback the transaction asynchronously by a background
+     thread. Thus rolling back a long transaction will not block server startup.
```

Comment:
Suggestion for better wording:

"This function should just change the transaction's state from prepared to active before returning. The actual rollback should then happen asynchroneously (eg. in a background thread). This way, rollbacks that take a long time to complete will not block server startup, and the database becomes available sooner to serve user queries."

Reply:
Hi Kristian, thanks for your suggestions. Fixed them all.